backend/src/main/java/com/company/project/entity/Timeslot.java

```diff
@@ -14,7 +16,8 @@
 @Table(name = "timetable")
 @NoArgsConstructor
 @AllArgsConstructor
-@Data
+@Getter
+@Setter
```

Comment:
Z ciekawości, coś tu się wywalało? Bo `@Data` jest aliasem do kilku - w tym `@Getter `i  `@Setter`

Reply:
Tak, lombok nie potrafił zapisać studenta  do slotu, i slotu do studenta

Reply:
Pytanie tylko, czy nie psuje się coś, bo `@Data` dodaje też `toString`, `hashCode` i `equals`. Ale jakk mówisz, że teraz ich dodaje, to chyba jest okej

Reply:
Sprawdzę to

Reply:
Ja uważam, że jak zrobisz testy w postmanie (ew. jakieś tymczasowe endpointy) i będzie działało, to można zostawić, ew. wrzuć tam `//TODO <jakiś komentarz>` żeby mieć oznaczone miejsce, które może być sus

Reply:
Ok

Reply:
Doczytałem. Głównym powodem przez który się psuło był fakt że Timeslot i Student miały odniesienie do siebie nawzajem. Powodowało to pętlę przy generowaniu hashcode. Dodałem adnotacje do konkretnie tych pól by nie brane były pod uwagę podczas liczenia hascode.